wp-content/plugins/core/src/Theme/Resources/Colors.php

```diff
@@ -33,4 +33,44 @@ public static function get_color_options_subset( array $colors = [] ) {
 		return array_intersect_key( self::get_color_options(), array_flip( $colors ) );
 	}
 
+	/**
+	 * Create an array of color options for use with the ACF Swatch field.
+	 *
+	 * @param array $colors
+	 *
+	 * @return array
+	 */
+	public static function get_color_options_for_acf( array $colors = [] ) {
+		$colors     = array_intersect_key( self::get_color_options(), array_flip( $colors ) );
+		$acf_colors = [];
+		foreach ( $colors as $color ) {
+			$acf_colors[ $color[ 'color' ] ] = $color[ 'label' ];
+		}
+
+		return $acf_colors;
+	}
+
+	/**
+	 * Return a color's key (I.e. - `black`) from the color's hex value (I.e. - `#000000`).
+	 *
+	 * Useful for getting a color's key name for use with CSS classes from the color's value
+	 * like when grabbing the value from an ACF swatch field.
+	 *
+	 * @param string $color
+	 *
+	 * @return string
+	 */
+	public static function get_color_key_from_acf_value( $color = '' ) {
+		$name = '';
+		foreach ( self::get_color_options() as $key => $value ) {
+			if ( in_array( $color, $value ) ) {
+				$name = $key;
```

Comment:
You could also just `return` the $key here instead of tossing in a `break`

Reply:
Good catch. Fixed.